logger_test.go

```diff
@@ -930,6 +930,17 @@ func TestConfigFromEnvOk(t *testing.T) {
 	}
 }
 
+func TestInvalidFile(t *testing.T) {
+	prev := jWriter.w
+	invalidFile := os.NewFile(^uintptr(0), "invalid-file")
+	jWriter.w = invalidFile
+	b := ConsoleLogging()
+	jWriter.w = prev
+	if b {
+		t.Errorf("expected not to be console logging")
+	}
+}
+
 // io.Discard but specially known to by logger optimizations for instance.
```

Comment:
```suggestion
// like io.Discard except io.Discard is checked for by logger optimizations and we want to avoid that.
```

Reply:
Clearer, but I still don't get the "checked for by logger"

for + by look awkward

Reply:
https://cs.opensource.google/go/go/+/refs/tags/go1.22.5:src/log/log.go;l=84

Reply:
added ^ to the comment